styles/error-page.scss

```diff
@@ -0,0 +1,58 @@
+@import './variables';
```

Comment:
File name should be camel case (errorPage.scss)

Reply:
Done